File: src/test/java/at/ac/uibk/dps/cirrina/data/DefaultDescriptions.java
package at.ac.uibk.dps.cirrina.data;

import org.junit.jupiter.api.Test;

public class DefaultDescriptions {

  public static final String complete = "pkl/complete/main.pkl";

  public static final String completeNested = "pkl/completeNested/main.pkl";

  public static final String invoke = "pkl/invoke/main.pkl";

  public static final String timeout = "pkl/timeout/main.pkl";

  public static final String pingPong = "pkl/pingPong/main.pkl";

  @Test
  void testLoadResource() {
    assert !complete.isEmpty();
    assert !completeNested.isEmpty();
    assert !invoke.isEmpty();
    assert !timeout.isEmpty();
    assert !pingPong.isEmpty();
  }
}

File: src/test/java/at/ac/uibk/dps/cirrina/data/DefaultDescriptions.kt
package at.ac.uibk.dps.cirrina.data

import at.ac.uibk.dps.cirrina.utils.TestUtils.resourceUri
import java.net.URI

object DefaultDescriptions {
  val complete: URI by lazy { resourceUri("pkl/complete/main.pkl") }
  val invoke: URI by lazy { resourceUri("pkl/invoke/main.pkl") }
  val timeout: URI by lazy { resourceUri("pkl/timeout/main.pkl") }
  val pingPong: URI by lazy { resourceUri("pkl/pingPong/main.pkl") }
  val noop: URI by lazy { resourceUri("pkl/noop/main.pkl") }
  val empty: URI by lazy { resourceUri("pkl/noop/empty.pkl") }
}
